src/test/groovy/ee/tuleva/onboarding/user/UserServiceSpec.groovy

```diff
@@ -23,4 +28,47 @@ class UserServiceSpec extends Specification {
     updatedUser.phoneNumber == "555555"
   }
 
+  def "can register a non member user as a member"() {
+    given:
+    def user = sampleUserNonMember().build()
+    userRepository.findOne(user.id) >> user
+    memberRepository.getMaxMemberNumber() >> 1000
```

Comment:
I personally would skip magic numbers and do `lastMemberNumber = 1000` and use the variable